packages/mgt-components/package.json

```diff
@@ -37,10 +37,10 @@
   },
   "dependencies": {
     "@microsoft/mgt-element": "*",
-    "@microsoft/microsoft-graph-client": "^2.2.1",
+    "@microsoft/microsoft-graph-client": "^3.0.0",
     "@microsoft/microsoft-graph-types": "^2.0.0",
     "@microsoft/microsoft-graph-types-beta": "^0.16.0-preview",
-    "@fluentui/web-components": "0.22.1",
+    "@fluentui/web-components": "^1.3.8",
```

Comment:
This should probably be targeting ^2.5.3

Might be that we want one of the other PRs for the fluent upgrade to merge first then update this PR from that state of next/fluentui

Reply:
Version bumped in this PR